query.txt

```diff
@@ -0,0 +1,8 @@
+declare namespace svg ="http://www.w3.org/2000/svg";
+
+let $rects := doc("./platforms.svg")//svg:svg/svg:rect
+return (
+  count($rects),
+  for $rect in $rects
+  return concat($rect/@x, " ", $rect/@y, " ", $rect/@width, " ", $rect/@height)
+)
```

Comment:
Put this file into a more appropriate place and document how to use it?

Reply:
Nah, to much of an effort. It also requires adding a check on reloading and stuff. Not worth it.